experimental/origin-dapp2/src/pages/user/User.js

```diff
@@ -1,13 +1,24 @@
 import React, { Component } from 'react'
 import { Query } from 'react-apollo'
 import get from 'lodash/get'
+import { fbt } from 'fbt-runtime'
 
 import IdentityQuery from 'queries/Identity'
 import Reviews from 'components/Reviews'
 import Avatar from 'components/Avatar'
 
 import UserListings from './_UserListings'
 
+function parseUserSocialID(attestations) {
+  const socialUrlObject = {}
+  attestations.forEach(attestation => {
+    const socialData = JSON.parse(attestation)
```

Comment:
I don't think it is necessary to parse. This should already be an object ?

Reply:
It actually returns as a string from IPFS, at least when I pull it from the `profile` object. It still has all the object structure but was a string.

Reply:
Ah... The graphQL client side probably returns this as a string rather than an Object.
Might be worth updating the GraphQL schema to return structured attestations data.
@nick any thoughts on this ?